src/resources/workflows/interfaces.ts

```diff
@@ -75,7 +75,7 @@ export interface Schedule {
   workflow: string;
   data: { [key: string]: any };
   last_occurrence_at: string | null;
-  next_occurrence_at: string;
+  next_occurrence_at: string | null;
   inserted_at: string;
   updated_at: string;
   repeats: ScheduleRepeatProperties[];
```

Comment:
Just to clarify, the `repeats` field will never be null. This field is set to `not null`  in the DB and has a default value of `[]`. 
So we know a schedule will always have repeats set as an empty list or as a list containing repeat rules.

PR for more context: https://github.com/knocklabs/switchboard/pull/1708